src/exec/Shelley.hs

```diff
@@ -5,11 +5,13 @@
 module Shelley where
 
 import qualified Codec.CBOR.Read as CBOR (DeserialiseFailure)
+import qualified Codec.Serialise as Serialise (decode, encode)
 import qualified Data.ByteString.Lazy as Lazy
 import qualified Data.Reflection as Reflection (given)
 import Data.Void (Void)
 import Network.Socket (SockAddr)
 
+import Cardano.Binary (fromCBOR, toCBOR)
```

Comment:
I already added these back in the base PR 044a1052046358947fc4062801f50b713e06d17d